src/uu/kill/src/kill.rs

```diff
@@ -164,12 +167,14 @@ fn print_signals() {
     }
 }
 
-fn list(arg: Option<&String>) -> UResult<()> {
-    match arg {
-        Some(x) => print_signal(x),
-        None => {
-            print_signals();
-            Ok(())
+fn list(signals: &Vec<String>) {
+    if signals.is_empty() {
+        print_signals();
+    } else {
+        for signal in signals {
+            if let Err(e) = print_signal(signal) {
+                uucore::show!(e)
+            }
```

Comment:
This looks like it prints the error, but then forgets to reflect it in the exit code. Is that the case?

Reply:
The `show!` macro sets the exit code properly.

Reply:
Oh! Yes, that makes sense, my bad. Awesome!